roles/polarion/defaults/main.yml

```diff
@@ -20,6 +20,6 @@
 
 cifmw_polarion_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}"
 cifmw_polarion_jump_repo_dir: "{{ cifmw_polarion_basedir }}/polarion-jump"
-cifmw_polarion_jump_result_dir: "{{ cifmw_polarion_basedir }}/tests/tempest/"
+cifmw_polarion_jump_result_dir: "{{ cifmw_polarion_basedir }}/tests/{{ cifmw_run_test_role | default('tempest') }}/"
 cifmw_polarion_jump_repo_branch: "master"
 cifmw_polarion_use_stage: false
```

Comment:
why don't we want to add the new var here?

cifmw_run_test_role: tempest

Reply:
that variable comes from the `run_test` role - not a good idea to redefine it here.